src/index/vespa_.py

```diff
@@ -104,71 +112,79 @@ class VespaFamilyDocument(BaseModel):
 
 
 def build_vespa_family_document(
-        task,
-        embeddings,
-        search_weights_ref,
-    ) -> VespaFamilyDocument:
+    task,
+    embeddings,
+    search_weights_ref,
+) -> VespaFamilyDocument:
     return VespaFamilyDocument(
-            search_weights_ref=search_weights_ref,
-            family_name=task.document_name,
-            family_name_index=task.document_name,
-            family_description=task.document_description,
-            family_description_index=task.document_description,
-            family_description_embedding=embeddings[0].tolist(),
-            family_import_id=task.document_metadata.family_import_id,
-            family_slug=task.document_metadata.family_slug,
-            family_publication_ts=task.document_metadata.publication_ts.isoformat(),
-            family_publication_year=task.document_metadata.publication_ts.year,
-            family_category=task.document_metadata.category,
-            family_geography=task.document_metadata.geography,
-            family_source=task.document_metadata.source,
-            document_import_id=task.document_id,
-            document_slug=task.document_slug,
-            document_languages=task.document_metadata.languages,
-            document_md5_sum=task.document_md5_sum,
-            document_content_type=task.document_content_type,
-            document_cdn_object=task.document_cdn_object,
-            document_source_url=task.document_metadata.source_url,
-        )
+        search_weights_ref=search_weights_ref,
+        family_name=task.document_name,
+        family_name_index=task.document_name,
+        family_description=task.document_description,
+        family_description_index=task.document_description,
+        family_description_embedding=embeddings[0].tolist(),
+        family_import_id=task.document_metadata.family_import_id,
+        family_slug=task.document_metadata.family_slug,
+        family_publication_ts=task.document_metadata.publication_ts.isoformat(),
+        family_publication_year=task.document_metadata.publication_ts.year,
+        family_category=task.document_metadata.category,
+        family_geography=task.document_metadata.geography,
+        family_source=task.document_metadata.source,
+        document_import_id=task.document_id,
+        document_slug=task.document_slug,
+        document_languages=task.document_metadata.languages,
+        document_md5_sum=task.document_md5_sum,
+        document_content_type=task.document_content_type,
+        document_cdn_object=task.document_cdn_object,
+        document_source_url=task.document_metadata.source_url,
+    )
 
 
 def build_vespa_document_passage(
-        family_document_id,
-        search_weights_ref,
-        text_block,
-        embedding
-    ) -> VespaDocumentPassage:
+    family_document_id,
+    search_weights_ref,
+    text_block,
+    text_block_window: str,
```

Comment:
only this one is typed of the arguments

Reply:
yes, I added the typing (it wasn't in the original code before i touched it). Trying to leave this as the same quality code as it exists in the pipeline rather than making too many updates as it'll just increase the size of the diff with that code later